geopyspark/geotrellis/render.py

```diff
@@ -145,3 +153,35 @@ def lookup(self, col, row, zoom=None):
         result = pngrdd.lookup(col, row)
 
         return [bytes for bytes in result]
+
+    def persist(self, storageLevel=StorageLevel.MEMORY_ONLY):
+        """Set the persistence mode of the underlying RDD structure.
+
+        Args:
+            storage_mode (StorageLevel): The storage mode to set.
+
+        Returns: The current object
+        """
+        self.is_cached = True
+
+        for level in self.pngpyramid:
+            level.persist(storageLevel)
```

Comment:
Python `StorageLevel` needs to be converted to an instance of java `StorageLevel` through 
```py
javaStorageLevel = self.geopysc.pysc._getJavaStorageLevel(storageLevel)
```